alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportGetFindingsAction.kt

```diff
@@ -167,7 +166,7 @@ class TransportGetFindingsSearchAction @Inject constructor(
                 )
                 val getMonitorResponse: GetMonitorResponse =
                     this@TransportGetFindingsSearchAction.client.suspendUntil {
-                        execute(GetMonitorAction.INSTANCE, getMonitorRequest, it)
+                        execute(AlertingActions.GET_MONITOR_ACTION_TYPE, getMonitorRequest, it)
```

Comment:
rewrite request to avoid serialization issue?

Reply:
This is following strategies of your previous refactoring PRs - see https://github.com/opensearch-project/alerting/pull/606/files

Could you explain more what the potential issue could be?

Reply:
https://github.com/opensearch-project/alerting/blob/main/alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportIndexMonitorAction.kt#L121-L123

we had faced ser/deserialization issues due to different plugins not sharing the same classloader amongst themselves

so, although these tests may pass now, the plugin invoking get monitor request would fail if the request is not transformed and re-deserialized in the transport layer